src/main/scala/org/ergoplatform/api/routes/InfoRoute.scala

```diff
@@ -73,4 +77,60 @@ object InfoRoute {
       "peersCount" -> connectedPeersLength.asJson
     ).asJson
   }
+
+
+  case class Info(
+                   name: String,
+                   version: String,
+                   headersHeight: Int,
+                   fullHeight: Int,
+                   bestHeaderId: Option[String],
+                   bestFullHeaderId: Option[String],
+                   previousFullHeaderId: Option[String],
+                   stateRoot: Option[String],
+                   difficulty: Long,
+                   unconfirmedCount: Int,
+                   stateType: String,
+                   stateVersion: Option[String],
+                   isMining: Boolean,
+                   votes: String,
+                   peersCount: Int
+                 )
+
+  object Info {
+
+    def make(nodeId: Array[Byte],
```

Comment:
`def apply` will also work and will allow for nicer syntax. `Info(…)` instead of `Info.make(…)`